### DISCUSSION TITLE
[Script request] Twenty.com

### DISCUSSION THREAD

**Discussion by chrisjenx** [community-scripts/ProxmoxVE#496] [Request script]
### Application Name

Twenty (Self Hosted CRM)

### Website

https://twenty.com/developers/section/self-hosting

### Description

Not super important, not sure how many people would benefit, but would be good to support business tools too, thinking more and more startups might be running there own clusters to be thrifty. I'll be trying this (setup manually of course) 

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by bbha01** [comment]
I would love to see this too!

**Reply by chrisjenx** [reply]
Especially as they update the dang thing like every week

**Comment by drVLecter** [comment]
Hi @MickLesk, 

the team at twentyhq upgraded to Node22 and confirmed on the following link:

[https://github.com/twentyhq/core-team-issues/issues/492](url)

Is this enough to remove the obstacle to move twenty.com script from Deferred into Backlog or Progress queue, please?

**Reply by tremor021** [reply]
https://github.com/twentyhq/core-team-issues/issues/492#issuecomment-2958541717

After this is done, it should be possible to take a look

**Reply by drVLecter** [reply]
Hi Slavisa @tremor021 , this seem to be upgraded. Does this qualify Twenty to get taken out of Deferred into Ready to pick or something more is missing?

**Comment by MickLesk** [comment]
Hello everyone,

I just wanted to remind you that this script has been in the DEV branch for testing for quite some time now. We’re getting almost no feedback, but we can see from the download numbers that users are installing it.
Please provide feedback. Otherwise, we’ll move it to the Stale Block next month and remove it again. 

Of course, I always ask for clean logs (systemctl - journalctl) whenever possible, or for console screenshots or exports for browser-related issues.

Link: https://github.com/community-scripts/ProxmoxVED/issues/1396

**Reply by chrisjenx** [reply]
Sure, will install this weekend when I have a moment, didn't know there was a dev branch to pull from :)

**Reply by chrisjenx** [reply]
Worked great! :shipit:, checked upgrade too 🥳
